index.html

```diff
@@ -249,6 +252,87 @@ <h2>Introduction</h2>
         </li>
       </ul>
 
+      <section class="informative">
+        <h3>Intended Audience</h3>
+
+        <p>
+The intended audience for the Verifiable Credentials Data Model 1.0 W3C Recommendation (Recommendation) 
+is a broad range of professionals interested in furthering
+the application of verifiable credentials technology for use in software apps, agents, and services.
+        </p>
+        <p>
+The primary audience includes software architects, application developers, and user experience (UX)
+specialists;
+as well as other people involved in a broad range of standards efforts related to decentralized identity,
+verifiable credentials, and secure storage.
+        </p>
+      </section>
+
+      <section class="informative">
+        <h3>Purpose</h3>
+
+        <p>
+The purpose of this Recommendation is to assist the intended audience by providing a deeper understanding
+of what a verifiable credential is as well as how to use verifiable credentials technology
+when designing and creating new software apps, agents, and services. 
+This also includes architects and developers creating reusable software libraries intended
+to assist architects and developers in incorporating verifiable credentials technology into their solutions.
+        </p>
+        <p>
+Verifiable credentials technology is adaptable to a wide variety of data serialization technologies
+(data platform bindings) including (but not limited to) JSON, JSON-LD, JWT, XML, POCOs (plain old CLR objects), etc.
+In this version of the Recommendation, documentation is provided (via the examples) to illustrate how
+JSON and JSON-LD data platform bindings can be used to support the creation, transmission, and verification
+of personal, business, governmental, and other types of data encapsulated in a verifiable credential.
```

Comment:
"transmission" may be over stepping here; it's a data model spec and "transmission" moves into protocol which was intentionally avoided and not in the charter when the spec was written.

Reply:
> transmission

I agree ...as a writing style, I like to include lists of things of length 3 ...they tend to be easier to read.  

Is there a third term we can use along with "creation" and "verification" in this sentence?

Reply:
"transformation" would work, given that you should be able to (in some cases) translate from one serialization to another.